client/js/keybinds.js

```diff
@@ -41,7 +41,7 @@ Mousetrap.bind([
 	"alt+up",
 	"alt+down",
 ], function(e, keys) {
-	const channels = sidebar.find(".chan");
+	const channels = sidebar.find(".chan").not(".network.collapsed .channel");
```

Comment:
This still allows queries and other windows to be targeted. This needs to target `:not(.lobby)` or something like that.